gobblin-data-management/src/test/java/org/apache/gobblin/data/management/copy/writer/FileAwareInputStreamDataWriterTest.java

```diff
@@ -481,4 +497,14 @@ public void cleanup() {
     }
   }
 
+  protected static class TestFileAwareInputStreamDataWriter extends FileAwareInputStreamDataWriter {
+
+    public TestFileAwareInputStreamDataWriter(State state, int numBranches, int branchId) throws IOException {
+      super(state, numBranches, branchId, null);
+    }
+    @Override
+    protected void setAclOnPath(FileSystem theFs, Path path, List<AclEntry> aclEntries) {
```

Comment:
aah... I see you do plan to override.

to be very clear: the original method absolutely, 100% deserves javadoc justifying what otherwise appears a superfluous abstraction.

Reply:
also, with an empty/no-op impl, there doesn't appear to be any way to validate test correctness.  do you also perceive the gap?

Reply:
I got rid of the stubbing altogether in my latest commit since I had to accommodate for changes related to manifest dataset and have updated the unit tests accordingly